action.yaml

```diff
@@ -55,20 +56,16 @@ runs:
           echo "Using Python: ${{ inputs.python_version }} 🐍"
         fi
 
-        # Handle path_prefix input consistently and when absent
-        path_prefix="${{ inputs.path_prefix }}"
-        if [ -z "$path_prefix" ]; then
-          # Set current directory as path prefix
-          path_prefix='.'
-        else
-          # Strip any trailing slash in provided path
-          path_prefix="${path_prefix%/}"
-        fi
-        # Verify is a valid directory path
+        # Verify path_prefix is a valid directory path
         if [ ! -d "$path_prefix" ]; then
           echo 'Error: invalid path/prefix to project directory ❌'; exit 1
         fi
-        echo "path_prefix=$path_prefix" >> "$GITHUB_ENV"
+
+        # Setup dedicated coverage report directory outside project path
+        coverage_dir="$RUNNER_TEMP/coverage_reports"
+        mkdir -p "$coverage_dir"
```

Comment:
This should ideally be done with a `mktemp -d` instead of hard coding the directory name since you export it into the environment anyway.

Reply:
@ModeSevenIndustrialSolutions I recommended using `mktemp` and you still ended up effectively hardcoding a path. Can you explain why?